### PR TITLE
Create 2021-10-05-str-methods.md

### PR DESCRIPTION
**Pull request by arjuntony**
New blog for String methods

### CODE REVIEW

**Review by asarfraaz — COMMENTED** [review]
Please give some examples for all the methods. That will help understand what the text means.

**Comment by asarfraaz** [comment]
Good to see the changes coming in. To make it more consistent, try using the same variable name in all examples. And for variable names, something like "ex" or "sent" would be more appropriate instead of "str".

**Review by asarfraaz — COMMENTED** [review]
All looks to be good now. Are there more changes expected? Or, shall I go ahead and merge the changes ?

**Comment by arjuntony** [comment]
@asarfraaz all done , we are good to merge if you are fine with the data

**Review by asarfraaz — APPROVED** [review]
Shall do the changes myself after the merge

File: _posts/2021-10-05-str-methods.md
Comment (asarfraaz) [reply _posts/2021-10-05-str-methods.md:433]: Not required. Existing example gives a better understanding. The initial empty string is more meaningful in this context.

**Pull request merged**
